### ISSUE TITLE
dealing with blank MF in the simple annoton editor

### ISSUE THREAD

**Issue by krchristie** [geneontology/noctua#492]
The simple annoton editor should **NOT** assume that it is appropriate to associate an anonymous MF with ECO:0000035 for "no biological data found". This evidence code is used specifically to indicate that a curator looked at **ALL** available literature and did not find anything, so it should definitely not be assumed that this evidence code is appropriate. 

I would be satisfied, for the moment, if it just didn't assume any evidence at all, because then I can use the canvas to clone appropriate experimental evidence so that the BP and/or CC annotations I really want to make will get generated, without having to also go through a series of clicks to delete the inappropriate ND evidence (that lacks the appropriate GO_REF and thus is non-functional anyway, with respect to allowing the associated BP and CC annotations of the annoton to be generated).

While on the subject of unknown MFs in Noctua, here are some additional comments:

1. At a recent Noctua call, we discussed the fact that there is a difference between when the data supports that the gene product is performing _some_ function versus when the curator wants to make a CC annotation WITHOUT assuming that the gene product is performing a function at that location. 

2. In addition, in practice as an annotator, I often just want to add new annotations from the paper I am reading, without taking the time to link up to previous work. The work described in PMID:27340223 is a great example. We already know that Dnah11 is a dynein heavy chain present in the ciliary axoneme and required for ciliary motility. However, that is prior knowledge for this paper and I don't really want to create an anonymous MF, since I know what it is. Associating the real evidence for the BP or CC term, and then filtering out the anonymous MF for the purpose of generating a GPAD is acceptable with respect to the current annotation paradigm and showing annotations on MOD websites such as MGI. However, when we are eventually displaying models, I don't really want the model to say this function is unknown. I realize that it is possible to put the known function and use TAS as the evidence, but I don't want to create these extra TAS annotations either, and definitely not to get new TAS annotations in the GPAD output.

Basically, I think we need to have multiple use cases for what is meant by blank MF:
- really unknown, ND evidence
- there is some MF involved, willing to have anonymous MF, but ND not appropriate
- MF is known from previous work, but don't want to put it in now (could use TAS from this paper, but GOC has also previously agreed to avoid using TAS, and I don't want to generate more TAS annotations)
- want to make a CC annotation WITHOUT saying anything about whether MF occurs there

Here's a relevant test model:

**ID** - gomodel:59b38c1100000155 
**Name** - TEST: assuming ND evidence, F-P link directionality - KRCX

**Comment by kltm** [comment]
Also tagging @cmungall .

**Comment by cmungall** [comment]
> really unknown, ND evidence

agreed. is there any action required to support this?

> there is some MF involved, willing to have anonymous MF, but ND not appropriate

agreed. already supported?

> MF is known from previous work, but don't want to put it in now (could use TAS from this paper, but GOC has also previously agreed to avoid using TAS, and I don't want to generate more TAS annotations)

this seems more of a curator consistency question. I would personally strongly prefer making the model complete. Having the information in separate models is formally weaker. The TAS guidelines were pre-GO-CAM. TAS seems very much in the spirit of making more complete models

> want to make a CC annotation WITHOUT saying anything about whether MF occurs there

Do we have this discussion in another ticket? Don't want to fork the discussion.

**Comment by cmungall** [comment]
> want to make a CC annotation WITHOUT saying anything about whether MF occurs there

OK, I made a ticket with hopefully precise instructions on how to implement this (see above). I think this should be straightforward.

**Comment by krchristie** [comment]
While making a bunch of new TAS annotations may be in the spirit of the models, it is very much NOT in the spirit of long standing guidelines for making sure we really know where the source of the evidence.  With respect to the GPAD output, I definitely do not want a bunch of new TAS annotations.

**Comment by vanaukenk** [comment]
Closing; we now have the standard annotation editor for curators.
